crates/echo-wasm-abi/README.md

```
@@ -0,0 +1,13 @@
+<!-- SPDX-License-Identifier: Apache-2.0 OR MIND-UCAL-1.0 -->
+<!-- © James Ross Ω FLYING•ROBOTS <https://github.com/flyingrobots> -->
+# echo-wasm-abi
+
+Shared WASM-friendly DTOs for Echo/JITOS living specs. Mirrors the minimal graph + rewrite shapes used in Spec-000 and future interactive specs.
+
+## Types
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)